src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectToolsCommandResolver.cs

```diff
@@ -76,14 +75,14 @@ public CommandSpec Resolve(CommandResolverArguments commandResolverArguments)
         }
 
         private CommandSpec ResolveCommandSpecFromAllToolLibraries(
-            IEnumerable<LibraryRange> toolsLibraries,
+            IEnumerable<ProjectLibraryDependency> toolsLibraries,
```

Comment:
Because `ProjectFile.Tools` are `ProjectLibraryDependency`, it was easier to do transformation to `LibraryRange` in this method.